docs/debug-checks.md

```diff
@@ -0,0 +1,9 @@
+# Debug Checks
+
+MvRx has a debug mode that enables some new checks. When you integrate MvRx into your app, you will need to create your own MvRxViewModel that extends BaseMvRxViewModel. One of the required things to override is protected abstract val debugMode: Boolean. You could set it to something like BuildConfig.DEBUG. The checks include:
```

Comment:
```suggestion
MvRx has a debug mode that enables some validation checks. When you integrate MvRx into your app, you will need to create your own MvRxViewModel that extends BaseMvRxViewModel. One of the required things to override is protected abstract val debugMode: Boolean. You could set it to something like BuildConfig.DEBUG. The checks include:
```

Reply:
`that extends BaseMvRxViewModel` - not true with my PR

Reply:
We can update this once your PR lands. We'll want to audit the flesh out the docs prior to releasing anyway